libs/s25main/network/GameServer.h

```diff
@@ -52,6 +52,8 @@ class GameServer :
     /// Assign players that do not have a fixed team, return true if any player was assigned.
     static bool assignPlayersOfRandomTeams(std::vector<JoinPlayerInfo>& playerInfos);
 
+    void SetNumPlayers(unsigned num);
```

Comment:
Maybe document as something like:
```suggestion
    /// Change number of players (after loading the map but before starting the game)
    void SetNumPlayers(unsigned num);
```